### ISSUE TITLE
app.example.toml seems redundant if you're preferring to ship this as Docker

### ISSUE THREAD

**Issue by Leopere** [JordanKnott/project-citadel#26]
Docker can just pull environment variables from your docker swarm stack file or even docker-compose.  Would it be unreasonable to ask for the `.toml` file to not be required?

**Comment by JordanKnott** [comment]
Once I work more on the config system, you'll be able to specify all options either through env variables or configs (which I prefer) - I'll probably work more on the config system once I get role enforcement finished

**Comment by tionis** [comment]
I would suggest as hybrid system in which each config option can either be specified in the toml or by specifying the whole coonfig variable path as environment variable or run argument.  An good example of this would be the way [oAuth2-Proxy](https://oauth2-proxy.github.io/oauth2-proxy/configuration#environment-variables) handles this.

**Comment by Leopere** [comment]
Sounds ideal to me.

**Comment by JordanKnott** [comment]
This has now been implemented as of commit c2ef8a7d565aa97907f5ae3ee61cc87f07127d64

For example, database settings can be set either through a config file or through the ENV variables

```
TASKCAFE_DATABASE_HOST
TASKCAFE_DATABASE_NAME
TASKCAFE_DATABASE_USER
TASKCAFE_DATABASE_PASSWORD
```
